application/templates/deployment.yaml

```
@@ -246,13 +246,10 @@ spec:
         {{- end }}
         {{- end }}
       {{- end }}
+        {{- if .Values.deployment.resources }}
         resources:
-          limits:
-            memory: {{ .Values.deployment.resources.limits.memory }}
-            cpu: {{ .Values.deployment.resources.limits.cpu }}
-          requests:
-            memory: {{ .Values.deployment.resources.requests.memory }}
-            cpu: {{ .Values.deployment.resources.requests.cpu }}
+{{ toYaml .Values.deployment.resources | indent 10 }}
```

Accompanying the `with` suggestion.

```suggestion
          {{- toYaml . | nindent 10 }}
```